18/README.md

```diff
@@ -2,211 +2,158 @@
+5. Each Validator is represented in `outputs`.
+6. Each entry in `outputs` can be spent by only one Validator, and the amount attached to it is equal to the power of that Validator.
+
+**Note: any change in the Validator Set will make old Elections invalid.**
 
 ### Extra: Vote delegation
 Vote delegation is trivial. Let's consider a Network of three Members: Alice, Bob, and Carly. Alice is the Initiator, and starts a new Election. Alice generates a `CREATE` transaction with three outputs, one per each Member. Bob wants to delegate his vote to Carly, so he transfers his output to Carly, granting her more votes she can spend in the way she wants.
 
-### Example: an Election to add a new Validator
-Alice (power `20`), Bob (power `10`), Carly (power `10`) and Daniel (power `10`) want to add Frank to the Network. Alice is the Initiator, and starts a new Election. Alice generates a `CREATE` transaction with `50` vote tokens. The transaction looks roughly like this:
-
-```json
-{
-  "asset": {
-    "data": {
-      "type": "election",
-      "name": "upsert-validator",
-      "version": "1.0",
-      "matter": "After the meeting we had on May 23rd, 2018, we decided to add Frank to the Network.",
-      "args": [
-        "Frank's public key",
-        "Frank's node id",
-	"Frank's hostname",
-	"New voting power"
-      ]
-    }
-  },
-  "id": "e...047",
-  "inputs": [
-    {
-      "fulfillment": "p...34C",
-      "fulfills": null,
-      "owners_before": [
-        "Alice's public key"
-      ]
-    }
-  ],
-  "metadata": null,
-  "operation": "CREATE",
-  "outputs": [
-    {
-      "amount": "20",
-      "condition": {
-        "details": {
-          "public_key": "Alice's public key",
-          "type": "ed25519-sha-256"
-        },
-        "uri": "ni:///sha-256;c...7-8Y?fpt=ed25519-sha-256&cost=131072"
-      },
-      "public_keys": [
-        "Alice's public key"
-      ]
-    },
-    {
-      "amount": "10",
-      "condition": {
-        "details": {
-          "public_key": "Bob's public key",
-          "type": "ed25519-sha-256"
-        },
-        "uri": "ni:///sha-256;b...123?fpt=ed25519-sha-256&cost=131072"
-      },
-      "public_keys": [
-        "Bob's public key"
-      ]
-    },
-    {
-      "amount": "10",
-      "condition": {
-        "details": {
-          "public_key": "Carly's public key",
-          "type": "ed25519-sha-256"
-        },
-        "uri": "ni:///sha-256;c...13C?fpt=ed25519-sha-256&cost=131072"
-      },
-      "public_keys": [
-        "Carly's public key"
-      ]
-    },
-    {
-      "amount": "10",
-      "condition": {
-        "details": {
-          "public_key": "Daniel's public key",
-          "type": "ed25519-sha-256"
-        },
-        "uri": "ni:///sha-256;c...7-8Y?fpt=ed25519-sha-256&cost=131072"
-      },
-      "public_keys": [
-        "Daniel's public key"
-      ]
-    }
-  ],
-  "version": "2.0"
-}
-```
+### Constraints
```

Comment:
I think, it makes sense to put this into the `About Election finality` section - the text there seems to be a perfect introduction into what we explain here.